src/utils/utils.cpp

```diff
@@ -1,9 +1,8 @@
 // thx https://github.com/jbro129/Unity-Substrate-Hook-Android
-
+#include "../../shared/utils/utils.h"
```

Comment:
why move this line?

Reply:
one's own header should always be the first to be included (if at all possible)

Reply:
agreed